spec/defines/rule/remote_spec.rb

```diff
@@ -2,15 +2,13 @@
 
 describe 'rsyslog::rule::remote' do
   context 'supported operating systems' do
-    on_supported_os.each do |os, os_facts|
+    on_supported_os.each do |os, facts|
```

Comment:
should not have been changed from os_facts to facts, as it makes the code difficult to understand (scoping)